protocol/messenger_communities.go

```diff
@@ -2300,7 +2300,11 @@ func (m *Messenger) RequestImportDiscordCommunity(request *requests.ImportDiscor
 			communityWithChats, changes, err := m.communitiesManager.CreateChat(discordCommunity.ID(), communityChat, false)
 			if err != nil {
 				m.cleanUpImport(communityID)
-				importProgress.AddTaskError(discord.ChannelsCreationTask, discord.Error(err.Error()))
+				errmsg := err.Error()
+				if err == communities.ErrInvalidCommunityDescriptionDuplicatedName {
```

Comment:
This although is valid, it's not a recommended practice:

```suggestion
				if errors.Is(err, communities.ErrInvalidCommunityDescriptionDuplicatedName) {
```